src/timeline/js/directives/misc.js

```diff
@@ -33,10 +33,8 @@ App.directive("ngRightClick", function ($parse) {
   return function (scope, element, attrs) {
     var fn = $parse(attrs.ngRightClick);
     element.bind("contextmenu", function (event) {
-      scope.$apply(function () {
```

Comment:
No need to apply the entire scope (for Angular) after calling a context menu. This was conflicting with another call to `$scope.$apply()`